### PR TITLE
Remove project logs access for Limited Guest role

### LINKED ISSUE #10438: Project log access for Limited Guest Role
The [Limited Guest Role](https://github.com/goharbor/harbor/blob/master/docs/user_guide.md#rbac) introduced in 1.10.0 specifies that

```
they cannot see logs or the other members of a project
```

However, project logs are still returned in `/api/logs`, as well as the "Logs" page in the UI.

### PR DESCRIPTION
**Pull request by mhuangpivotal**
Fixes #10438

### CODE REVIEW

File: src/core/api/log.go [hunk 1]
```diff
@@ -87,17 +89,21 @@ func (l *LogAPI) Get() {
 			return
 		}
 
-		if len(projects) == 0 {
+		ids := []int64{}
+		for _, project := range projects {
+			roles := l.SecurityCtx.GetProjectRoles(project.ProjectID)
+			if len(roles) > 0 && roles[0] != common.RoleLimitedGuest {
```
Comment (heww) [inline src/core/api/log.go]: You should use `HasProjectPermission` from `BaseController` to check the user has logs permission.

Comment (heww) [reply src/core/api/log.go]: Hi, mhuangpivotal, can you resolve the problem I replied asap so that this fix can be released with 1.10.1 ?

Comment (mhuangpivotal) [reply src/core/api/log.go]: Hi @heww, thanks for the review. I have pushed the requested change.

Comment (heww) [reply src/core/api/log.go]: @mhuangpivotal Thank you very much.

The swagger-codegen-cli-2.3.1.jar download failed from the http://central.maven.org which cause the ci failed and today a PR fix this issue merged in release-1.10.0.

Can you rebase this PR with release-1.10.0 branch? Thanks!

Comment (mhuangpivotal) [reply src/core/api/log.go]: @heww done

**Pull request merged**